lib/src/fhir_rest_client.dart

```diff
@@ -9,6 +9,32 @@ class FhirRestClient {
 
   FhirRestClient({required this.dio, required this.baseUrl});
 
+  Future<List<String>> getSchema() async {
```

Comment:
I would renamed this function to Future<Map<String,dynamic>> getCapabilityStatement() and have the schema extraction in the caller, then it is something that can be re-used across applications.

Reply:
563f058